CMakeLists.txt

```diff
@@ -19,16 +19,31 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
                       ${CMAKE_SOURCE_DIR}/cmake/macros)
 
 include(Options)
+# If we were given a python executable, we'll extract the name from that to use in
+# the python shebang replacement. This will make these files more portable by not
+# having some full path to a python executable which may not exist on another machine.
+if (DEFINED PYTHON_EXECUTABLE)
+    get_filename_component(PYTHON_EXE_BASENAME ${PYTHON_EXECUTABLE} NAME)
+endif()
+
```

Comment:
Is there a particular reason why this set here in the middle of these includes? Naively I'd think this would go down next to where `PXR_PYTHON_SHEBANG` is set, so that we'd use whatever `PYTHON_EXECUTABLE` comes out of the Packages include.